### ISSUE TITLE
Avalonia replaces spaces in the solution name, but doesn't change the path as well, so it can't find its own files no longer

### ISSUE THREAD

**Issue by Yveske1966** [AvaloniaUI/Avalonia#13747]
## Describe the bug
The picture below says it all. The path itself remains without spaces, while internally Avalonia expects underscores.

## To Reproduce
Make a new solution with spaces in its name and try to compile

## Expected behavior
Avalonia should either refrain from making changes or change the path as well.

## Screenshots
![My program with spaces](https://github.com/AvaloniaUI/Avalonia/assets/68047793/37e9400a-5def-4bc0-b7b1-eb4870923ecc)

## Environment 
- OS: Windows 11 Pro 22H2
- PC: AMD Ryzen 9 3950X 16-Core Processor
- RAM: 64,0 GB
- VS; Microsoft Visual Studio Community 2022 (64-bit) - Preview Version 17.9.0 Preview 1.0
- Avalonia-Version: 11.0.5

## Additional context
None

**Comment by MrJul** [comment]
Please note that Avalonia itself doesn't care about the spaces. Nor does MSBuild. I believe it's an issue with the project templates, where the spaces were replaced with underscores at creation.

Can you please confirm that you were using the `avalonia.xplat` template? If yes, I'll move this issue to the templates repository.

In the meantime, you should be able to replace the underscores with spaces in the .sln and .csproj and get everything to compile (it might be faster to simply create a project without spaces from the templates though).

**Comment by Yveske1966** [comment]
I added the Avalonia extension to Visual Studio and created a new project with the "Avalonia Cross Platform Application (AvaloniaUI) template. As I'm new to this, I don't know if that matches avalonia.xplat.

**Comment by MrJul** [comment]
Yes, that's the one. I can clearly reproduce the problem with it.
I'm moving this issue to the templates repo, thanks.
